src/main/kotlin/com/zepben/evolve/services/common/extensions/IdentifiedObjectExtensions.kt

```diff
@@ -33,6 +29,7 @@ internal fun IdentifiedObject.validateReference(other: IdentifiedObject, getter:
 
 /**
  * Removes all names that belong to this object from their corresponding [NameType][com.zepben.evolve.cim.iec61970.base.core.NameType].
+ * TODO: investigate whether this is superseded by clearNames()
  */
 fun IdentifiedObject.removeNamesFromTypes() {
     names.forEach { it.type.removeName(it) }
```

Comment:
What was the original purpose of this function that removes an io's names from their types but doesn't remove them from the io?

Reply:
It is used in the gis network extractor when the io is going to be unlinked and removed from the network anyway, so I suppose it's meant to be a faster(?) way to mark the io and its names as unreachable and garbage-collectable

Reply:
Removed due to limited usage.